server/src/infra/repositories/person.repository.ts

```diff
@@ -40,6 +40,18 @@ export class PersonRepository implements IPersonRepository {
     return result.affected ?? 0;
   }
 
+  async removeFaceFromPerson(oldPersonId: string, newPersonId: string, assetId: string): Promise<number> {
+    const result = await this.assetFaceRepository
+      .createQueryBuilder()
```

Comment:
Can you rewrite this without the query builder as a normal update?

Can we call this reassign face instead?